docs/content/faq.md

```diff
@@ -22,6 +22,20 @@ See the [remote setup docs](/remote_setup/) for more info.
 
 This has now been documented in its own [remote setup page](/remote_setup/).
 
+### How can I get rid of the "Config file not found" notice?
+
+If you see a notice like 'NOTICE: Config file "rclone.conf" not found', this
+means you have not configured any remotes.
+
+If you need to create a remote, see the [remote setup page](/remote_setup/).
```

Comment:
I think this would be a better link https://rclone.org/docs/#configure

(Without the `https://rclone.org` prefix for the docs, but you've figured that out already :-)